lib/MetaCPAN/Types/Internal.pm

```diff
@@ -93,16 +95,27 @@ coerce Profile, from HashRef,
 subtype Tests,
     as Dict [ fail => Int, na => Int, pass => Int, unknown => Int ];
 
-subtype BugSummary,
+subtype BugsRT,
     as Dict [
     (
         map { $_ => Optional [Int] }
-            qw(new open stalled patched resolved rejected active closed)
+            qw( active closed new open patched rejected resolved stalled )
     ),
-    type   => Str,
     source => Str
     ];
 
+subtype BugsGithub,
```

Comment:
I like the custom types.  Could we rename them to `GitHubBugs` and `RTBugs`?  That would be in keeping with types like `PositiveNum` where the adjective precedes the noun.  I think that reads more like a natural language.  (Also, note the casing on `GitHub`).